AdobeStockAsset/Test/Integration/SaveAssetKeywordInsertIgnoreTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeStockAsset\Test\Integration;
+
+use Magento\AdobeStockAsset\Model\ResourceModel\Keyword;
+use Magento\AdobeStockAssetApi\Api\Data\KeywordInterface;
+use Magento\Framework\App\ResourceConnection;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+class SaveAssetKeywordInsertIgnoreTest extends TestCase
+{
+    /**
+     * @var Keyword
+     */
+    private $keywordResourceModel;
+
+    /**
+     * @var ResourceConnection
+     */
+    private $resourceConnection;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
+    {
+        $this->keywordResourceModel = Bootstrap::getObjectManager()->get(Keyword::class);
+        $this->resourceConnection = Bootstrap::getObjectManager()->get(ResourceConnection::class);
+        $this->resourceConnection->getConnection()->beginTransaction();
```

Comment:
There is no need to use a transaction for rollback as the populated data is known.

Please use @magentoDataFixture to specify a fixture. The fixture file name + `_rollback` suffix will identify the file containing a rollback algorithm for corresponding fixture